app/src/app.py

```diff
@@ -29,6 +29,8 @@ def create_app() -> APIFlask:
 
     configure_app(app)
     register_blueprints(app)
+    build_index_route(app)
```

Comment:
nit: maybe call it `register_index` or `register_index_route` to be consistent with `register_blueprints`